redash/__init__.py

```diff
@@ -53,8 +54,11 @@ def create_redis_connection():
             redis_db = redis_url.path[1]
         else:
             redis_db = 0
-
-        r = redis.StrictRedis(host=redis_url.hostname, port=redis_url.port, db=redis_db, password=redis_url.password)
+        # Redis passwords might be quoted with special characters
+        redisPassword = ""
+        if redis_url.password is not None:
+            redisPassword = urllib.unquote(redis_url.password)
+        r = redis.StrictRedis(host=redis_url.hostname, port=redis_url.port, db=redis_db, password=redisPassword)
```

Comment:
We use `snake_case` and not `camelCase`. And this can be simplified to:

```python
redis_password = redis_url.password and urllib.unquote(redis_url.password)
```

If the password is `None`, it will just assign `None`.